### ISSUE TITLE
Allow usage of cert-manager provided x509 ecdsa key as ssh key for the shepherd

### ISSUE THREAD

**Issue by fionera** [monogon/monogon#195]
Currently we generate a ssh-key and store it on a pvc. We should allow the usage of an existing cert-manager provided secret to make deployment easier

**Comment by fionera** [comment]
Yes we have to as this is the ssh key thats used by the shepherd to connect to the machines

**Comment by q3k** [comment]
Yes, I am aware of that. In theory we could recover from this automatically by detecting that a machine has been provisioned without the current SSH key of the shepherd - but I'm not sure if it's worth it.
